labs/lab1/chapter1.md

```diff
@@ -1,17 +1,17 @@
-# LAB 1: podman
+# LAB 1: podman and buildah
 
 In this lab we will explore the podman environment. If you are familiar with podman this may function as a brief refresher. If you are new to podman this will serve as an introduction to podman basics. Don't worry, we will progress rapidly. To get through this lab, we are going to focus on the environment itself as well as walk through some exercises with a couple of podman images/containers to tell a complete story and point out some things that you might have to consider when containerizing your application.
 
-What is [podman](https://github.com/projectatomic/libpod), you may ask? Well, the README explains it in detail but, in short, it is a tool for manipulating containers created by docker or other tools (such as buildah). The docker utility provides build, run, and push functions on docker containers. We are leveraging three tools, which support multiple kinds of containers, that do each function separately. Namely, [buildah](https://github.com/projectatomic/buildah) for building, podman for download, verification, and run, and [skopeo](https://github.com/projectatomic/skopeo) for distribution (pushing).
+What is [podman](https://github.com/projectatomic/libpod), you may ask? Well, the README explains it in detail but, in short, it is a tool for manipulating OCI compliant containers created by docker or other tools (such as buildah). The docker utility provides build, run, and push functions on docker containers via a docker daemon. We are leveraging three daemonless tools, which support OCI compliant containers, that do each function separately. Namely, [buildah](https://github.com/projectatomic/buildah) for building, podman for download, verification, and run, and [skopeo](https://github.com/projectatomic/skopeo) for distribution (pushing). You don't need the overhead of running a daemon to leverage these capabilities.
```

Comment:
we could probably beat the "daemonless" drum a little less hard right here. I'm not trying to come off sounding too biased.

Reply:
also, should we even mention skopeo? podman transparently uses the tech in the background right? do we ever call skopeo directly in the lab?

Reply:
I didn't realize podman wrapped skopeo too. I was trying to point out the unix-philosophy of it.